python/lsst/cp/pipe/ptc/cpSolvePtcTask.py

```diff
@@ -955,22 +1070,23 @@ def fitPtc(self, dataset):
             ptcFitType = dataset.ptcFitType
         else:
             raise RuntimeError("ptcFitType is None of empty in PTC dataset.")
-        matrixSide = self.config.maximumRangeCovariancesAstier
-        nanMatrix = np.empty((matrixSide, matrixSide))
-        nanMatrix[:] = np.nan
+        # For FULLCOVARIANCE model fit
+        matrixSideFit = self.config.maximumRangeCovariancesAstierFullCovFit
```

Comment:
Isn't this going to be used for all model types?  If this is used for all model types, what do we gain by changing all of these variable names?

Reply:
Just for the `FULLCOVARIANCE` model type.